### DISCUSSION TITLE
Which Host Type for Setting Up Anycubic Kobra 2 Pro in Orca Slicer?

### DISCUSSION THREAD

**Discussion by Taacccoo** [OrcaSlicer/OrcaSlicer#6178] [Q&A]
Hi Orca Community,

I recently purchased an Anycubic Kobra 2 Pro and I’m trying to set it up in Orca Slicer. I’m a bit confused about which host type I should select for the wireless connection.

Does anyone know which host type is required for the Anycubic Kobra 2 Pro? Is it possible?

Any detailed guidance or step-by-step instructions would be greatly appreciated!

Thanks in advance for your help!

**Comment by docdawning** [comment]
I suspect it's just not going to be working out with Orca. But I hope someone who actually knows can comment. My Kobra 2 Max has 1 port open as far as I can see that is 8888, but that doesn't seem to work with Orca. A small sort of saving-grace is that the new slicer "Anycubic Slicer Next" is based on Orca, so that should be a familiar experience. Maybe somehow due to that it can be figured out how to port over whatever settings and/or code in to Orca?
